crates/goose-cli/src/commands/configure.rs

```diff
@@ -38,6 +44,46 @@ pub async fn handle_configure(provided_profile_name: Option<String>) -> Result<(
     Ok(())
 }
 
+async fn set_systems(additional_systems: &mut Vec<AdditionalSystem>) -> Result<(), Box<dyn Error>> {
```

Comment:
i'd probably remove this from the configure command entirely i think? you've figured out how here but its kind of awkward to implement a dialogue looping around additions

I'd suggest at the end of configure we say "we've added the developer system by default, to add or remove systems, use \n goose system {add|remove} systemurl"

Reply:
then we'd need a new subcommand